packages/mgt-chat/src/components/ChatListItem/ChatListItem.tsx

```diff
@@ -174,25 +206,45 @@ export const ChatListItem = ({ chat, myId, onSelected }: IMgtChatListItemProps &
   const removeHtmlPTags = (str: string) => {
     return str.replace(/<\/?p>/g, '');
   };
+  const removeHtmlATags = (str: string) => {
+    return str.replace(/<\/?a[^>]*>/g, '');
+  };
 
   const enrichPreviewMessage = (previewMessage: NullableOption<ChatMessageInfo> | undefined) => {
     let previewString = '';
+    let content = previewMessage?.body?.content as string;
+
+    // handle emojis
+    content = rewriteEmojiContent(content);
+
+    // handle images
+    const graphImageUrlRegex = /<img[^>]+?src=["']https:\/\/graph\.microsoft\.com\/[^"'>]+?["']/;
+
+    const imageMatch = content.match(graphImageUrlRegex);
+    if (imageMatch) {
+      content = 'Sent an image.';
+    }
 
     // handle general chats from people and bots
     if (previewMessage?.from?.user?.id === myId) {
-      previewString = 'You: ' + previewMessage?.body?.content;
+      previewString = 'You: ' + content;
     } else if (previewMessage?.from?.user?.displayName) {
-      previewString = previewMessage?.from?.user?.displayName + ': ' + previewMessage?.body?.content;
+      previewString = previewMessage?.from?.user?.displayName + ': ' + content;
     } else if (previewMessage?.from?.application?.displayName) {
-      previewString = previewMessage?.from?.application?.displayName + ': ' + previewMessage?.body?.content;
+      previewString = previewMessage?.from?.application?.displayName + ': ' + content;
     }
 
     // handle all events
     if (previewMessage?.eventDetail) {
-      previewString = previewMessage?.body?.content as string;
+      previewString = content as string;
     }
 
-    return removeHtmlPTags(previewString);
+    // removes <p> tags from the string
+    previewString = removeHtmlPTags(previewString);
```

Comment:
It seems like this should happen on content not necessarily previewString and be moved up above the "handle general chats" section.